sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_util.py

```diff
@@ -155,8 +154,7 @@ def process(self, elem):
           json_dict[FieldNames.CREDIT_CARD],
           json_dict[FieldNames.CITY],
           json_dict[FieldNames.STATE],
-          nexmark_json_util.millis_to_timestamp(
-              json_dict[FieldNames.DATE_TIME]),
+          nexmark_model.millis_to_timestamp(json_dict[FieldNames.DATE_TIME]),
```

Comment:
nit: millis_to_timestamp can stay in this file, it's not causing a circular dep.